torchvision/prototype/features/_feature.py

```diff
@@ -132,13 +117,11 @@ def resized_crop(
         interpolation: InterpolationMode = InterpolationMode.BILINEAR,
         antialias: bool = False,
     ) -> Any:
-        # Just output itself
-        # How dangerous to do this instead of raising an error ?
         return self
 
-    def pad(self, padding: List[int], fill: int = 0, padding_mode: str = "constant") -> Any:
-        # Just output itself
-        # How dangerous to do this instead of raising an error ?
+    def pad(
+        self, padding: List[int], fill: Union[int, float, Sequence[float]] = 0, padding_mode: str = "constant"
```

Comment:
Why don't we support `Sequence[int]` as well? Did you face JIT-scriptability issues related to how it handles seq of ints and floats? If we add support, we need to put them in all places.

Reply:
I think we can add `Sequence[int]` to the type hint (I didn't add it as it starts looking very bulky). JIT is not concerned as `pad` is not scriptable (in general torch script does not recognize `Sequence` and we should map it to `List`).
In following PR, we have to make same type hint for all `fill` usages.

Reply:
Sounds good. You wanna put a TODO on the code or you keep track of this elsewhere?

Reply:
I'll be dealing with this in Transforms PR (next one, not yet sent)